static/sass/partials/_header.scss

```diff
@@ -45,8 +163,43 @@ header {
   }
 }
 
-@media (min-width: 900px) {
+@media (min-width: 992px) {
```

Comment:
Please add a small comment to describe what kind of device this breakpoint is targeting.

Reply:
done